cmd/admin-heal.go

```diff
@@ -99,10 +101,12 @@ func (u healObjectMessage) JSON() string {
 // checkAdminHealSyntax - validate all the passed arguments
 func checkAdminHealSyntax(ctx *cli.Context) {
 	if len(ctx.Args()) != 1 {
-		cli.ShowCommandHelpAndExit(ctx, "heal", 1) // last argument is exit code
+		cli.ShowSubcommandHelp(ctx)
```

Comment:
That's why `cli.ShowCommandHelpAndExit(ctx, "heal", 1)` wasn't working in the first place. I'll open a new bug in the cli upstream. Meanwhile, probably I'll reform the structure of admin heal command and make it similar to what we do with policy command.